decentraland/helpers/index.js

```diff
@@ -221,6 +221,14 @@ export const restoreCreationStateOfAccountFromBlockchain = async account => {
     creationActions[FUNDING_WITH_MANA] = null;
   }
 
+  if (fundedWithMana || !marketplaceWasApproved) {
```

Comment:
Hmm, this `OR` seems dangerous to me. Maybe we should use an `AND` instead? Is there a specific case you want to catch with the `OR`?

Reply:
Oh, logic error. Fixed. Thanks!

Reply:
Thanks. Resolving this.